### ISSUE TITLE
Draft Non-Rider Facing Comms Plan

### ISSUE THREAD

**Issue by vykster** [cal-itp/benefits#896]
Splitting issue https://github.com/cal-itp/benefits/issues/650 into two. 

**Other**

- [x] https://github.com/cal-itp/benefits/issues/903
- [ ] https://github.com/cal-itp/benefits/issues/992
- [x] Host Non-rider facing Comms Brainstorm (scheduled 9/6/22)
- [x] Coordinate with other content owners (CalSTA, Caltrans, CDT, CDT D-ID, Login.gov, MST)
- [ ] https://github.com/cal-itp/benefits/issues/857
- [ ] #1298
- [x] Draft press release from CalSTA
- [x] Review from Login/gov comms team, addition of additional messaging, content, or channels to support login.gov team’s goals.

Resources:
- [Cal-ITP Benefits Launch Comms Plan](https://docs.google.com/document/d/11wfslytJJ02zb-u8fsY4D92CiG8EU9DXWUENeqnPMtY/edit#heading=h.ecujg6dssk7t)
- https://github.com/cal-itp/benefits/issues/842

## Acceptance Criteria

- [ ]

**Comment by machikoyasuda** [comment]
This ticket seems related https://github.com/cal-itp/benefits/issues/857

**Comment by thekaveman** [comment]
@MarikoLochridge proposal: all of the tickets mentioned here could be added to the [`Outreach` milestone](https://github.com/cal-itp/benefits/milestone/29) (I think you created that? Nice!) and then this ticket could be closed. It's basically serving as a milestone / collection of tasks.

**Comment by MarikoLochridge** [comment]
That's a great idea @thekaveman 

I added this ticket and all tickets mentioned to the milestone of outreach. 

I also converted this incomplete task 'Update fact sheet with login.gov information: Cal-ITP Benefits Fact Sheet #1298' to an issue  and assigned it to @srhhnry but uncertain if she was the correct person for this issue. If it is not Sarah let me know who is best to assign that issue to!
